src/cpp/config.cpp

```diff
@@ -0,0 +1,60 @@
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+extern "C" {
+  #include "config.h"
+}
+
+using namespace std;
```

Comment:
Including namespaces is a bad practice and I think we should avoid this. Our code should remain easily maintainable. If one day we happen to need something from other namespace then we should have to change lots of lines. [https://stackoverflow.com/questions/1452721/why-is-using-namespace-std-considered-bad-practice](url)

Reply:
I would argue it is only a problem if we would use another namespace like boost.

Anyways, if you want you can make the changes on this branch yourself :+1:

Reply:
Fixed in ae5ea3e3acb7a2f5c0fe74755608110cb96c3936.